samples/test-runner/build.gradle.kts

```diff
@@ -51,7 +51,7 @@ instrumentation {
         useInMemoryReport.set(true)
         useService.set(false) // todo fix broken
         fetchLogcatForIncompleteTests.set(true)
-        saveTestArtifactsToOutputs.set(true)
+        saveTestArtifactsToOutputs.set(false)
```

Comment:
This experiment doesn't work at my local machine. I got an exception: `can't mkdir`

Reply:
@dsvoronin I need your help to fix it)

Reply:
i will fix it myself, thanks!